manifests/helm/templates/deployment.yaml

```
@@ -0,0 +1,125 @@
+apiVersion: apps/v1  
+kind: Deployment  
+metadata:  
+  name: {{ include "go-broker.fullname" . }}  
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)